src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -63,6 +65,8 @@
   public static final Duration DEFAULT_META_QUERY_TIMEOUT = Duration.ofMillis(5000);
   public static final String DEFAULT_AUTH_PROTOCOL = "";
   public static final long DEFAULT_SESSION_RESET_SECS_WINDOW = 30;
+  public static final int DEFAULT_MAX_FETCH_COUNT = 100;
+  public static final int DEFAULT_MAX_FETCH_SIZE = 1000000; // bytes
```

Comment:
Better to naming `DEFAULT_MAX_FETCH_BYTES`, and assign `1 * 1024 * 1024`.

Reply:
ok